### PR TITLE
[PTFE-692] Change registry to ghcr

### PR DESCRIPTION
**Pull request by gaspardmoindrot**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -24,20 +25,23 @@ jobs:
...
 
-      - name: Login to Registry
+      - name: Login to ghcr
         uses: docker/login-action@v1
         with:
-          registry: registry.scality.com
-          username: ${{ secrets.REGISTRY_LOGIN }}
-          password: ${{ secrets.REGISTRY_PASSWORD }}
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
```
Comment (tcarmet) [inline .github/workflows/release.yml:32]:
```suggestion
          username: ${{ github.actor }}
```

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -24,20 +25,23 @@ jobs:
...
         uses: docker/build-push-action@v2
         with:
           context: .
           file: ./Dockerfile
           push: true
-          tags: "registry.scality.com/artifacts/artifacts:${{ github.event.inputs.tag }}"
+          build-args: |
+            VERSION=${{ github.event.inputs.tag }}
+          tags: |
+            ghcr.io/scality/bert-e:${{ inputs.tag }}
```
Comment (tcarmet) [inline .github/workflows/release.yml:44]:
```suggestion
            ghcr.io/scality/artifacts:${{ inputs.tag }}
```

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -24,20 +25,23 @@ jobs:
...
 
       - name: Build and push
         uses: docker/build-push-action@v2
         with:
           context: .
           file: ./Dockerfile
           push: true
-          tags: "registry.scality.com/artifacts/artifacts:${{ github.event.inputs.tag }}"
+          build-args: |
+            VERSION=${{ github.event.inputs.tag }}
```
Comment (tcarmet) [inline .github/workflows/release.yml:42]:
```suggestion
            VERSION=${{ inputs.tag }}
```

File: charts/artifacts/values.yaml [hunk 4]
```diff
@@ -49,11 +49,11 @@ cloudStorage:
 image:
   ## regitry to pull from.
   ##
-  registry: registry.scality.com
+  registry: ghcr.io
```
Comment (tcarmet) [inline charts/artifacts/values.yaml:52]: Please confirm that all the tagged images have been migrated to ghcr.io/scality/artifacts` and that we can pull the image without any kind of authentication.

**Review by tcarmet — CHANGES_REQUESTED** [review]

File: charts/artifacts/values.yaml (hunk 4 above)
Comment (gaspardmoindrot) [reply charts/artifacts/values.yaml:52]: Yes everything's good

File: .github/workflows/release.yml [hunk 5]
```diff
@@ -24,20 +25,23 @@ jobs:
...
 
       - name: Build and push
         uses: docker/build-push-action@v2
         with:
           context: .
           file: ./Dockerfile
           push: true
-          tags: "registry.scality.com/artifacts/artifacts:${{ github.event.inputs.tag }}"
+          build-args: |
+            VERSION=${{ inputs.tag }}
```
Comment (tcarmet) [inline .github/workflows/release.yml:42]: Why did you add this build argument? Just noticed it wasn't set before.

**Review by tcarmet — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 5 above)
Comment (gaspardmoindrot) [reply .github/workflows/release.yml:42]: I saw it in another project that was doing the same thing, and I found it interesting to copy it as well. Do you want me to remove it?

Comment (tcarmet) [reply .github/workflows/release.yml:42]: It's only useful if a proper `ARG VERSION` is defined in the `Dockerfile` like:

```dockerfile
ARG VERSION
```
And later on used by the service itself.

In this is case it is not set and even if set, it will not be used by the service so I see no point in keeping it:
```suggestion
```

**Review by tcarmet — APPROVED** [review]
